airflow/utils/email.py

```diff
@@ -254,6 +254,10 @@ def send_mime_email(
             smtp_password = airflow_conn.password
         except AirflowException:
             pass
+    if smtp_user is None:
```

Comment:
There will be no option to define smtp in airflow.cfg once https://github.com/apache/airflow/pull/30531 is completed.

@hussein-awala has plans to refacor this area of the code

Reply:
@eladkal noted and thanks for the info.